templates/community.html

```diff
@@ -127,5 +127,21 @@
             </div>
         </div>
     </a>
+    <a class="card community-card" href=" https://mastodon.social/@bevy">
+        <div class="card-image">
+            <img src="/assets/itchio-textless.svg" class="centered-card-image community-icon" alt="itch.io logo" />
```

Comment:
http://127.0.0.1:1111

Reply:
Yes, i where can i find it? i haven't changed because i don't know if it exists into the project

Reply:
Can i use an image from the internet? If  so, where do i add it into the project?

Reply:
Maybe grab the "white mark-only" svg from here: https://joinmastodon.org/branding.

You can place is alongside the other icons in this repository.